index.html

```diff
@@ -102,6 +102,17 @@ <h3>Lere</h3>
           </p>
         </article>
 
+        <article class="member">
+          <h3>Jewels</h3>
+          <p>Languages: HTML, CSS</p>
+          <p>GitHub: <a href="https://github.com/ShalomiAhavah">https://github.com/ShalomiAhavah</a></p>
+          </p>
```

Comment:
Please remove the paragraph tags that you are not using. Thanks.

Reply:
Done(: